### PR TITLE
Added option: Omit mandatory check for Importer

### PR DESCRIPTION
**Pull request by Pitu-pl**
Added new option `Omit mandatory check` for Importer.
We can select that we want to ignore checking mandatory fields in Object.

### CODE REVIEW

File: src/ImportDefinitionsBundle/Model/DefinitionInterface.php [hunk 1]
```diff
@@ -188,6 +188,16 @@ public function getStopOnException();
      */
     public function setStopOnException($stopOnException);
 
+    /**
+     * @return mixed
```
Comment (dpfaffenbauer) [inline src/ImportDefinitionsBundle/Model/DefinitionInterface.php]: use bool please

File: src/ImportDefinitionsBundle/Model/DefinitionInterface.php [hunk 2]
```diff
@@ -188,6 +188,16 @@ public function getStopOnException();
...
+     */
+    public function getOmitMandatoryCheck();
+
+    /**
+     * @param boolean $omitMandatoryCheck
```
Comment (dpfaffenbauer) [inline src/ImportDefinitionsBundle/Model/DefinitionInterface.php]: use bool please

File: src/ImportDefinitionsBundle/Model/Definition.php [hunk 3]
```diff
@@ -410,6 +415,22 @@ public function setStopOnException($stopOnException)
         $this->stopOnException = $stopOnException;
     }
 
+    /**
+     * @return bool|mixed|null
```
Comment (dpfaffenbauer) [inline src/ImportDefinitionsBundle/Model/Definition.php]: use following please

```
/*
 * {@inheritdoc}
 */
```

File: src/ImportDefinitionsBundle/Model/Definition.php [hunk 4]
```diff
@@ -410,6 +415,22 @@ public function setStopOnException($stopOnException)
...
+     */
+    public function getOmitMandatoryCheck()
+    {
+        return $this->omitMandatoryCheck;
+    }
+
+    /**
+     * @param boolean $omitMandatoryCheck
```
Comment (dpfaffenbauer) [inline src/ImportDefinitionsBundle/Model/Definition.php]: use following please

```
/*
 * {@inheritdoc}
 */
```

File: src/ImportDefinitionsBundle/Importer/Importer.php [hunk 5]
```diff
@@ -311,6 +311,11 @@ private function importRow(DefinitionInterface $definition, $data, $params, $fil
         }
 
         $object->setUserModification(0); //Set User to "system"
+
+        if ($definition->getOmitMandatoryCheck()) {
+            $object->setOmitMandatoryCheck(true);
+        }
+
```
Comment (dpfaffenbauer) [inline src/ImportDefinitionsBundle/Importer/Importer.php]: you could actually use this as well:

```php
$object->setOmitMandatoryCheck($definition->getOmitMandatoryCheck());
```

**Comment by Pitu-pl** [comment]
@dpfaffenbauer thanks for comments, I'm fixed all issues.

**Comment by dpfaffenbauer** [comment]
awesome thanks :)

**Pull request merged**